modules/host/storageobligations.go

```diff
@@ -1,14 +1,32 @@
 package host
 
-// TODO: Need to set up the multi-drive thing.
-
-// TODO: File contracts actually cannot be mutable to add fees - makes the file
-// contract id mutable too. BUT, revisions can be mutable.
-
 // storageobligations.go is responsible for managing the storage obligations
 // within the host - making sure that any file contracts, transaction
 // dependencies, file contract revisions, and storage proofs are making it into
 // the blockchain in a reasonable time.
+//
+// NOTE: Currently, the code partially supports changing the storage proof
+// window in file contract revisions, however the action item code will not
+// handle it correctly. Until the action item code is improved (to also handle
+// byzantine situations where the renter submits prior revisions), the host
+// should not support changing the storage proof window, especially to further
+// in the future.
+
+// TODO: Need to queue the action item for checking on the submission status of
+// the file contract revision. Also need to make sure that multiple actions are
+// being taken if needed.
+
+// TODO: Document mistakes that got made when constructing the Sia consensus
+// code. The first is not putting timeouts on keys including signatures? The
+// second was using floating point values in consensus code. The third is that
+// you can't build a concise storage proof (e.g. 0 bytes) for a file contract
+// with a size of 0.
```

Comment:
will clean this up over the next few days. There's clearly a lot of room for improvement in the host code and structure right now, but I'm trying to get it feature-complete before worrying too much about organization.